rtsp_test.go

```diff
@@ -202,7 +201,7 @@ func TestRTSPCameraPerformance(t *testing.T) {
 		timeoutCtx, timeoutCancel := context.WithTimeout(context.Background(), time.Second*10)
 		defer timeoutCancel()
 		config := resource.NewEmptyConfig(camera.Named("foo"), ModelAgnostic)
-		config.ConvertedAttributes = &Config{Address: "rtsp://" + h.s.RTSPAddress}
+		config.ConvertedAttributes = &Config{Address: "rtsp://" + h.s.RTSPAddress + "/"}
```

Comment:
Did we confirm that this doesn't double append `/`?

Reply:
Yeah I think that was a log that I inserted into gorstplib that was logging expected route and route request.

Confirmed the full RTSP URI was indeed rtsp://127.0.0.1:3251/

Moved to using `/stream1` in test cases to make things more clear (not having an empty route).